db/migrate/20220810094914_create_like.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] The piece of code that wrote will create a`user_id` but in the diagram, it should be `author_id` in order to achieve that try the following:
```RUBY
t.references :author, foreign_key: { to_table: 'users' }
```

Reply:
Thank you so much @youmari, definitely missed this bit of information 🙌. I will apply these changes in no time ✔.